api/data_refinery_api/views.py

```diff
@@ -845,6 +845,28 @@ class ProcessorJobList(generics.ListAPIView):
 # Statistics
 ###
 
+class AboutStats(APIView):
+    """ Returns general stats for the site, used in the about page """
+    @method_decorator(cache_page(10 * 60))
+    def get(self, request, version, format=None):
+        result = {
+            # count the total number of samples that are processed or that have a quant.sf file associated with them
+            'samples_available': Sample.objects.filter(is_processed=True).count() \
+                + Sample.objects.filter(is_processed=False, technology='RNA-SEQ', results__computedfile__filename='quant.sf').distinct().count(),
+            'total_size_in_bytes': OriginalFile.objects.aggregate(total_size=Sum('size_in_bytes'))['total_size'],
+            # count organisms with qn targets or that have at least one sample with quant files
+            'supported_organisms': Organism.objects.filter(qn_target__isnull=False).count() \
+                +  Organism.objects.filter(
+                    qn_target__isnull=True,
+                    sample__is_processed=False,
+                    sample__technology='RNA-SEQ',
+                    sample__results__computedfile__filename='quant.sf'
+                  ).distinct().count(),
```

Comment:
A similar comment about breaking this up. Combining the full calculation of these stats into a single expression makes them less readable I think.